targets/mbed/source/makejerry.cmake

```diff
@@ -1,4 +1,4 @@
-# Copyright 2015-2016 Samsung Electronics Co., Ltd.
+# Copyright 2015 Samsung Electronics Co., Ltd.
```

Comment:
Why? Downgrading the copyright date is odd to me. This file is touched by @akiss77 in 3ba286f3